plugin.cpp

```diff
@@ -36,35 +36,52 @@ static const char *default_config = QUOTE({
 			"type": "boolean",
 			"displayName": "Enabled",
 			"default": "false",
-			"order" : "5"
+			"order" : "7"
 		       	},
+        "toleranceMeasure": {
+			"description": "Whether tolerance is specified as a percentage or an absolute value",
```

Comment:
Description "Tolerance measure to be used either a percentage or an absolute value" makes more sense